src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -561,6 +561,27 @@ def create_app_package(self) -> None:
                 )
             )
 
+    def _expand_script_templates(
+        self, env: jinja2.Environment, jinja_context, scripts: List[str]
+    ):
+        queued_queries = []
```

Comment:
what's a query in this context? The same doesn't sound very meaningful here. Isn't it just script content? I know the name was there before, but it's a good opportunity to clarify.

Reply:
updated.